commit 2e6ac45f831cf174f6597d8b1e61e48fe6e245e1
Author: Marsell Kukuljevic <marsell@joyent.com>
Date:   2019-04-12T19:18:52+00:00 (6 months ago)
    
    TRITON-1392 Invalid params test fails when in test mode due to server_uuid
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/test/machines.test.js b/test/machines.test.js
index 38c52e85a..a9da6fc6e 100644
--- a/test/machines.test.js
+++ b/test/machines.test.js
@@ -585,16 +585,12 @@ test('Create machine with invalid parameters', function (t) {
         image: IMAGE_UUID,
         package: SDC_256.name,
         // Underscore will make name invalid:
-        name: '_a' + uuid().substr(0, 7),
-        // Obviously, not a valid UUID, but we don't want to notify customers
-        // about this:
-        server_uuid: '123456'
+        name: '_a' + uuid().substr(0, 7)
     };
 
     CLIENT.post('/my/machines', obj, function (err, req, res, _body) {
         t.ok(err, 'POST Create machine with invalid parameters');
-        t.ok(/name/.test(err.message));
-        t.notOk(/server/.test(err.message));
+        t.ok(err.message, 'Invalid name');
         t.end();
     });
 });
@@ -1903,6 +1899,7 @@ test('Create Machine using "params.bootrom" without "bhyve"', function (t) {
     });
 });
 
+
 test('teardown', function (t) {
     common.deletePackage(CLIENT, SDC_256, function (err) {
         common.deletePackage(CLIENT, SDC_256_INACTIVE, function (err2) {
